packages/components/src/components/timeline/timeline.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+import Text from '../text';
+
+type TOval = {
+    children: number;
+};
+
+const Oval = ({ children }: TOval) => {
+    return (
+        <div className='dc-timeline__oval'>
+            <Text color='colored-background' size='s' weight='bold' className='dc-timeline__number'>
+                {children}
+            </Text>
+        </div>
+    );
+};
+
+type TItem = React.HTMLAttributes<HTMLDivElement> & { item_title: string | React.ReactNode };
+
+type TTimeline = React.HTMLAttributes<HTMLDivElement> & { disabled_items: number[] };
```

Comment:
@yauheni-kryzhyk-deriv I suggest moving all type declarations to the top of the file🙂